### DISCUSSION TITLE
404 file not found

### DISCUSSION THREAD

**Discussion by ANGELCJR** [community/community#72836] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

so im trying to access my site from github pages, i understand the index.html is supposed to be in the root of my directory, right now its inside a public folder and i need it to be in a public folder becuse its a static file. this is the error i got

404
File not found
The site configured at this address does not contain the requested file.
If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

here is what my directory looks like

![github](https://github.com/community/community/assets/79959345/7025bac9-e816-4114-b860-4d414976b525)

![12123dcsdc](https://github.com/community/community/assets/79959345/49384d04-be5d-4dd5-9d1b-5c68d98e1bfe)

what should i do for my public folder to be seen?

**Comment by ktauchathuranga** [comment]
hiiii,
watch this video and follow the instructions
https://www.youtube.com/watch?v=M5mg0r4ajt4&pp=ygUfaG93IHRvIGhvc3QgYSB3ZWJzaXRlIG9uIGdpdGh1Yg%3D%3D
and also is your repo "public"?
